xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_support.xsd

```diff
@@ -114,9 +114,29 @@ Rail transport, Roads and Road transport
 	<xsd:element name="WheelchairVehicleRef" type="WheelchairVehicleRefStructure" substitutionGroup="VehicleEquipmentRef">
 		<xsd:annotation>
 			<xsd:documentation>Reference to a WHEELCHAIR VEHICLE EQUIPMENT.</xsd:documentation>
+			<xsd:documentation>DEPRECATED</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
 	<xsd:complexType name="WheelchairVehicleRefStructure" abstract="false">
+		<xsd:annotation>
+			<xsd:documentation>DEPRECATED</xsd:documentation>
```

Comment:
1. Documentation should not be only DEPRECATED
2. @trurlurl do we need structures in EPIAP, that are now changed or deprecated. If so there might be a problem.